numba/dispatcher.py

```diff
@@ -784,6 +784,8 @@ class ObjModeLiftedWith(LiftedWith):
     def __init__(self, *args, **kwargs):
         self.output_types = kwargs.pop('output_types', None)
         super(LiftedWith, self).__init__(*args, **kwargs)
+        if not self.flags.force_pyobject:
+            raise ValueError("expecting `flags.enable_pyobject`")
```

Comment:
Should that be ``expecting `flags.force_pyobject` `` ? and does the force imply enabled, this making the next check (on `self.flags.enable_pyobject`) redundant?

Reply:
Good catch. Fixed